app/views/ad_reminder_letters/bulk.html.erb

```diff
@@ -1,27 +0,0 @@
-<!doctype html>
```

Comment:
What was this file originally doing? It looks like a template for the reminder letters but was that not done through notify?

Reply:
Yes, that's done through Notify now. It was previously done within the Rails application, and this was also missed during the cleanup when Notify was adopted.